src/ostorlab/runtimes/local/runtime.py

```diff
@@ -190,13 +208,8 @@ def scan(
             The scan object.
         """
         try:
-            console.info("Creating scan entry")
-            if assets is None:
-                assets_str = "N/A"
-            else:
-                assets_str = f'{", ".join([str(asset) for asset in assets])}'
-                # TODO(mohsinenar): we need to add support for storing multiple assets and rename this to target.
-            self._scan_db = self._create_scan_db(asset=assets_str[:255], title=title)
+            if hasattr(self, "_scan_db") is False or self._scan_db is None:
+                self.prepare_scan(title=title, assets=assets)
```

Comment:
Why do we need to check hasattr, this hint at a bug somewhere.

Reply:
the init was mocked in some unit tests, i did address that.